wallet-new/docs/updates.md

```diff
@@ -252,37 +254,40 @@ data UpdateSeverity = Mandatory
                     | Optional
 ```
 
-For hard forks updates (or security-critical issues) we might want to set the severity
-to `Mandatory`, for soft forks and other non-critical updates we can label an update as `Optional`.
+For security-critical issues we might want to set the severity
```

Comment:
This should spawn a task in https://iohk.myjetbrains.com/youtrack/issue/CS-113

Reply:
Done! https://iohk.myjetbrains.com/youtrack/issue/CSL-2012